packages/uikit-react-native/src/domain/groupChannel/component/GroupChannelMessageList.tsx

```diff
@@ -53,19 +85,164 @@ const GroupChannelMessageList = (props: GroupChannelProps['MessageList']) => {
     },
   );
 
+  const onScrolledAwayFromBottom = useFreshCallback((value: boolean) => {
+    scrolledAwayFromBottomRef.current = value;
+    props.onScrolledAwayFromBottom(value);
+  });
+
   const scrollToBottom = useFreshCallback(async (animated = false) => {
     if (props.hasNext()) {
       props.onUpdateSearchItem(undefined);
-      props.onScrolledAwayFromBottom(false);
+      onScrolledAwayFromBottom(false);
 
       await props.onResetMessageList().catch((_) => {});
-      props.onScrolledAwayFromBottom(false);
+      onScrolledAwayFromBottom(false);
       lazyScrollToBottom({ animated });
     } else {
       lazyScrollToBottom({ animated });
     }
   });
 
+  const onPressUnreadMessagesFloatingCloseButton = useCallback(() => {
+    updateHasSeenNewLine(true);
+    updateHasUserMarkedAsUnread(false);
+    props.resetNewMessages?.();
+    confirmAndMarkAsRead([props.channel]);
+  }, [updateHasSeenNewLine, updateHasUserMarkedAsUnread, props.channel.url, props.resetNewMessages]);
+
+  const getPrevNonSilentMessage = useCallback(
+    (messages: SendbirdMessage[], prevMessageIndex: number): SendbirdMessage | null => {
+      if (messages.length <= prevMessageIndex) {
+        return null;
+      }
+
+      const prevMessage = messages[prevMessageIndex];
+      if (prevMessage) {
+        if (prevMessage.silent) {
+          return getPrevNonSilentMessage(messages, prevMessageIndex + 1);
+        } else {
+          return prevMessage;
+        }
+      }
+      return null;
+    },
+    [],
+  );
+
+  const findUnreadFirstMessage = useFreshCallback((isNewLineExistInChannel: boolean) => {
```

Comment:
nit: 요 이름이 좀 더 어울릴 것 같슴다 ㅎㅎ
```suggestion
  const findFirstUnreadMessage = useFreshCallback((isNewLineExistInChannel: boolean) => {
```

Reply:
변경했습니다!